songs_app_backend/songs/serializers.py

```diff
@@ -0,0 +1,108 @@
+from rest_framework import serializers
+from .models import Song, UserSongLike, UserSongFavorite, UserSongComment
+from taggit.serializers import TagListSerializerField, TaggitSerializer
+from .tasks import add_song_task
+from datetime import datetime, timedelta
+import pytz
+
+
+class SongSerializer(TaggitSerializer, serializers.ModelSerializer):
+    tags = TagListSerializerField()
+
+    class Meta:
+        model = Song
+        fields = ('id', 'title', 'singer', 'tags')
+
+
+class AddSongSerializer(TaggitSerializer, serializers.ModelSerializer):
+    tags = TagListSerializerField()
+
+    class Meta:
+        model = Song
+        fields = ('id', 'title', 'singer', 'tags', 'created_time')
+        extra_kwargs = {
+            'created_time': {'required': True}
+        }
```

Comment:
use the auto_now_add field in models so you don't have to cater for created_time